### DISCUSSION TITLE
Unban

### DISCUSSION THREAD

**Discussion by RTG15** [pterodactyl/panel#5076] [Support]
hey its roei i got a banned from 15 min i  thing so basicly i used script for a panel i didnt knowed its unallowed to do that im very sorry and my friend tom got banned too he just helped me i dont know why they got ban two for 1 message only sorry again i didndt know and i mean that i not have a something to lie to this just i want you know you said two 5 times you warning me i didnt see it only 1 message only from someone in server says NO SUPPORT and idk what a conutine of a that message i rellay sorry sory sorry if you can unban me or temp ban me week mounth it well better but not forvever this server is relleay inporant me and you helophelped me guys very time and i never used again please unban me sorry again

**Comment by rowansc1** [comment]
@RTG15 whats your Discord ID/username?
